Dockerfile.dapper

```diff
@@ -1,14 +1,26 @@
 FROM registry.suse.com/bci/golang:1.19
 
+# k3d and kubectl versions must be aligned with the Kubernetes versions
+# set in tests/k3s-bench-test.yaml.
+#  k3d is used for e2e tests and is not shipped on the final image.
+ARG K3D_VERSION=5.4.8
+ARG KUBERNETES_VERSION=1.28.0
+
+ENV GOLANGCI_LINT v1.51.2
+
 ARG DAPPER_HOST_ARCH
 ENV ARCH $DAPPER_HOST_ARCH
-ENV GOLANGCI_LINT v1.51.2
 
-RUN zypper -n install git docker vim less file curl wget
+RUN zypper -n install git docker vim less file curl wget awk jq
 RUN if [[ "${ARCH}" == "amd64" ]]; then \
         curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s ${GOLANGCI_LINT}; \
     fi
 
+RUN curl -s https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | TAG="v${K3D_VERSION}" bash
+ADD --chown=root:root --chmod=755 \
+    "https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/${ARCH}/kubectl" \
```

Comment:
As per this [PR](https://github.com/rancher/security-scan/pull/170), The GCS bucket will be deprecated in the future.
Could you replace:
`https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/${ARCH}/kubectl`  
with 
`https://dl.k8s.io/release/v${KUBERNETES_VERSION}/bin/linux/${ARCH}/kubectl`

Reply:
Nice catch - made the change. PTAL